### ISSUE TITLE
(app request) hardware info / system analysis / benchmarking

### ISSUE THREAD

**Issue by drandreaskrueger** [flatpak/flatpak#1294]
_This might be the wrong place to ask this question, then please advise where I can ask it. Thx._

## FlatPak: HardwareInfo & Benchmark tools?
We got a new Acer without Windows but with EndlessOS preinstalled. It took a while to realize that EndlessOS has castrated standard installation tools (`apt-get`, etc.) - but it supports FlatPak.

Now the search is on:
* a hardware info package = that tells us that all hardware is fine, and functioning; and incl GPU.
* a benchmark package = not as urgent, but would be nice to see how the machine compares.

Before we install anything else (a real Linux, Windows for gaming, etc) we would like to first test the hardware, so that -worst case scenario- we can return the laptop to the vendor.

Are you aware of any project with that focus?
I have looked at all the apps on https://flathub.org/ and the selection is short but good.
However, nothing about hardware info / system analysis / benchmarking.

### TL;DR: 
Which FlatPak project are you aware of with **hardware** focus?
Or: If you are a FlatPak expert, wouldn't it be a cool and rewarding challenge **to create one**?

Thanks a lot
Andreas

**Comment by drandreaskrueger** [comment]
crossposted to https://community.endlessos.com/t/flatpak-software-wishlist/1657/53?u=andreaskrueger

**Comment by TingPing** [comment]
This is certainly not the place for package requests, the endless os forum is probably ideal (especially since as you mentioned that is an endless specific limitation).
